src/iupvs01.f

```diff
@@ -110,3 +110,5 @@ RECURSIVE FUNCTION IUPVS01(LUNIT,S01MNEM) RESULT(IRET)
 902   CALL BORT('BUFRLIB: IUPVS01 - A MESSAGE MUST BE OPEN IN INPUT '//
      . 'BUFR FILE, NONE ARE')
       END
+
+      
```

Comment:
Just curious, why the two added blank lines at the end of this file?

Reply:
Inadvertent.